crates/bevy_remote/src/builtin_methods.rs

```diff
@@ -1017,4 +1242,37 @@ mod tests {
             entity: Entity::from_raw(0),
         });
     }
+
+    #[test]
+    fn reflect_export_test() {
```

Comment:
It'd be great if we had some more tests for different kinds of types (e.g. enums, tuples, etc.).

Would also be helpful if one of the tests also tested the JSON output (i.e. compared the entire output to a string). This also helps to ensure that everything not only behaves correctly, but _looks_ right too (and it's nice for for future devs to see what the result is actually supposed to look like).

Reply:
Agree, I will try to add some more tests.